js/atree.js

```diff
@@ -473,7 +475,7 @@ const atree_collect_spells = new (class extends ComputeNode {
                 case 'add_spell_prop': {
                     const { base_spell, target_part = null, cost = 0, behavior = 'merge'} = effect;
                     if (base_spell !== base_spell_id) { continue; }   // TODO: redundant? if we assume abils only affect one spell
-                    ret_spell.cost += cost;
+                    if ('cost' in ret_spell) { ret_spell.cost += cost; }
```

Comment:
is cost decremental on upgrades? 

just a sanity check

Reply:
on upgrades cost is input as negative if its reduced ye
~~it works on my machine~~